src/shared/config/routes.ts

```diff
@@ -4,5 +4,8 @@ export const ROUTES = {
     SUBCATEGORY: '/:category/:subcategory',
     RECIPE: '/:category/:subcategory/:id',
     THE_JUICIEST: 'the-juiciest',
+    SIGN_IN: 'sign-in',
+    SIGN_UP: 'sign-up',
+    VERIFICATION: 'verification',
```

Comment:
```suggestion
    SIGN_IN: '/sign-in',
    SIGN_UP: '/sign-up',
    VERIFICATION: '/verification',
```

почему так изначально не сделать?